### DISCUSSION TITLE
virtio-fs as a supported mount driver?

### DISCUSSION THREAD

**Discussion by jacksongoode** [abiosoft/colima#309] [Ideas]
Seeing the improvements that Docker Desktop has made with integrating this as the file sharing protocol I was wondering if there are plans to add this? I believe it's integrated into qemu, maybe a matter of exposing?

**Comment by abiosoft** [comment]
It is currently only supported for Qemu on Linux AFAIK.

**Reply by jacksongoode** [reply]
@abiosoft Oh so it's an option that can be specified in colima's template for example?

**Reply by abiosoft** [reply]
Not at all, it is not yet supported on Colima. I am only pointing out that even if it were to be supported, it will be limited to Linux.

**Comment by abiosoft** [comment]
Some good news! This is looking like a possibility https://github.com/lima-vm/lima/issues/889.

**Reply by abiosoft** [reply]
This has been added upstream and it is being added to Colima. 
This should bring parity to Docker Desktop with filesystem access speed.
